pkg/features/features.go

```diff
@@ -19,13 +19,14 @@ func init() {
 }
 
 const (
-	// ProcessStatusSync supports synchronization between process message channel statuses
-	ProcessStatusSync featuregate.Feature = "ProcessStatusSync"
+	// RequireAuthorization Supports application access authorization from edge sides
+	// alpha: v1.12
+	RequireAuthorization featuregate.Feature = "requireAuthorization"
```

Comment:
```suggestion
	// owner: @vincentgoat
	// alpha: v1.12

	// RequireAuthorization Supports application access authorization from edge sides
```

Reply:
the comment is less accurate，please add more detail

Reply:
done